### PR TITLE
SW-1121-dataset-saving

### PR DESCRIPTION
**Pull request by farfaraji**

### CODE REVIEW

File: codejail/jail_code.py [hunk 1]
```diff
@@ -376,3 +379,21 @@ def create_rlimits(effective_limits):
...
 
     return rlimits
+
+
+def save_artifacts(artifacts, save_path):
+    datasets_dest_dir = os.path.join(save_path, 'datasets')
+    os.mkdir(datasets_dest_dir)
+    os.chmod(datasets_dest_dir, 0o777)
+    for artifact_path in artifacts:
+        if artifact_path.endswith('.csv'):
```
Comment (antoineKorbit) [inline codejail/jail_code.py]: Why do we only take care of csv files ?
What about json, xlsx, img ?

File: codejail/jail_code.py [hunk 2]
```diff
@@ -376,3 +379,21 @@ def create_rlimits(effective_limits):
...
+            path = datasets_dest_dir
+        else:
+            continue
+        with open(artifact_path, 'r') as file:
+            content = file.read()
+            new_file = os.path.join(path, os.path.basename(artifact_path))
+            print(new_file)
```
Comment (antoineKorbit) [inline codejail/jail_code.py]: Let's remove this print statement.

**Review by antoineKorbit — COMMENTED** [review]
Overall looks good to me, just have a question on the artifacts types ?

Also please remove the print statement.

File: codejail/jail_code.py (hunk 1 above)
Comment (farfaraji) [reply codejail/jail_code.py]: so yeah we can add others but we need to know what folders need to be put into do we even wanna support xlsx , should we support .png and put them inside images folder

Comment (antoineKorbit) [reply codejail/jail_code.py]: Oh, I don't think it matters now, let's accept everything so if a need arrive to use image it will work automatically ?

**Review by antoineKorbit — APPROVED** [review]

**Pull request merged**